### PR TITLE
Lists + Text + Toast Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch consists of pages for the Lists, Text and Toast Components.

### CODE REVIEW

File: Component-Page/component-page.css [hunk 1]
```diff
@@ -41,6 +41,18 @@ h1.sub-heading {
...
+    justify-content: center;
+    overflow-x: scroll;
+    background-color: var(--code-example-bg);
+    color: silver;
+    border-radius: 5px;
+    margin-top: -1rem;
+    margin-bottom: 3rem;
+    padding: 1rem 0; */
+
+
```
Comment (romabulani) [inline Component-Page/component-page.css]: You can remove the comments if not required

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Review by Hnh013 — APPROVED** [review]
looks good to merge , classes are good and and optimized

File: Component-Page/component-page.css (hunk 1 above)
Comment (tejas-muthya37) [reply Component-Page/component-page.css]: Thanks for pointing this out. Done.

**Pull request merged**